pkg/apis/core/types.go

```diff
@@ -3358,9 +3362,16 @@ type PodStatus struct {
 	// give the resources on this node to a higher priority pod that is created after preemption.
 	// +optional
 	NominatedNodeName string
+
+	// HostIP address of the host to which the pod is assigned. Empty if not yet scheduled.
 	// +optional
 	HostIP string
 
+	// HostIPs holds the IP addresses allocated to the host. If this field is specified, the 0th entry must
+	// match the hostIP field. This list is empty if no IPs have been allocated yet.
```

Comment:
For both `empty` comment. It is empty if the pod has not started yet. A pod can be assigned to a node that has a problem in kubelet which in turns mean that HostIP/HostIPs will not be updated even if there is node is assigned to pod